src/pages/index.js

```diff
@@ -7,7 +7,6 @@ import Layout from '../components/Layout';
 import useFetch from '../hooks/useFetch';
 
 export default function HomePage() {
-
 	const {data, error, loading} = useFetch('api/teaser');
```

Comment:
For the possibility to use better variable names you should better use an array instead of a object in your hook useFetch or you name your variables with more specific names when you destructuring your object.

Example:
```suggestion
	const [teaser, error, loading] = useFetch('api/teaser');
```

or

```suggestion
	const {data as teaser, error, loading} = useFetch('api/teaser');
```

Reply:
Solution results in an uniterable object and Solution two get's marked as false syntax and I can't get it running.